distribution/src/dbscripts/mysql_transaction_count.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS CURRENT_STATS (
+  TIME_STAMP DATE NOT NULL,
+  NODE_ID VARCHAR(40) NOT NULL,
+  TRANSACTION_COUNT INT NOT NULL,
+  TRANSACTION_COUNT_ENCRYPTED TEXT NOT NULL
```

Comment:
shall we make this into VARCHAR? Better if we can keep it in a dynamically allocated type